db/migrate/20200728054557_refer_config_proxy_to_http_proxy.rb

```diff
@@ -0,0 +1,17 @@
+class ReferConfigProxyToHttpProxy < ActiveRecord::Migration[6.0]
+  def change
+    ::ForemanVirtWhoConfigure::Config.find_each do |config|
+      if config.proxy.present?
+        create_http_proxy_if_not_exist(config)
+      end
+    end
+  end
+
+  private
+
+  def create_http_proxy_if_not_exist(config)
+    http_proxy = HttpProxy.find_by(url: config.proxy)
+
+    HttpProxy.create!(name: "virt_who_#{config.proxy}", url: config.proxy) unless http_proxy
```

Comment:
@jturel What should I do if the config has not valid proxy url?

Reply:
IMO we should use create instead of create! so we don't error here. Last thing we want to do is break upgrades. Logging an error would be fine if we were unable to create the record. Hopefully there's already some kind of validation on config.proxy so we can be reasonably sure that the URL is valid!

Reply:
@jturel As we are changing `proxy` field to `http_proxy_id`, so if config proxy url is not valid, we will not create any HttpProxy   object for that proxy. Can we keep nil for `http_proxy_id` in such cases?

Reply:
I think that's OK. If it didn't have a valid URL then it's likely that configuration was not working. Do you have any concerns about doing it?

Reply:
In a configuration right now, can we enter just an IP address? I wonder if that will prevent the HttpProxy from being saved. Might have to convert it to a url...